models/iaf_cond_alpha_mc_kinetics.h

```diff
@@ -0,0 +1,651 @@
+weights must be positive numbers!
+
+Depending on value of active flag, calcium spike maybe triggered depending on its kinetics dynamics.
+The spike is modeled using first order dynamics, whereby
+
+ICa_ = M_CA * H_CA * G_Ca * (E_Ca - V_m.d)
+d/dt M_CA = (minf - M_CA) / tau_m
+d/dt H_CA = (hinf - H_CA) / tau_h
+
+and
+
+minf = 1.0/( 1.0 + exp ^ ( (V_m.d - half_m) * -slope_m ) )
+hinf = 1.0/( 1.0 + exp ^ ( (V_m.d - half_h) * -slope_h ) )
+
+
+Parameters:
+The following parameters can be set in the status dictionary. Parameters
+for each compartment are collected in a sub-dictionary; these sub-dictionaries
+are called "soma", "proximal", and "distal", respectively. In the list below,
+these parameters are marked with an asterisk.
+
+V_m*         double - Membrane potential in mV
+E_L*         double - Leak reversal potential in mV.
+C_m*         double - Capacity of the membrane in pF
+E_ex*        double - Excitatory reversal potential in mV.
+E_in*        double - Inhibitory reversal potential in mV.
+g_L*         double - Leak conductance in nS;
+tau_syn_ex*  double - Rise time of the excitatory synaptic alpha function in ms.
+tau_syn_in*  double - Rise time of the inhibitory synaptic alpha function in ms.
+I_e*         double - Constant input current in pA.
+t_L*         double - Leak during refractory period in nS
+nt_L*        double - Leak at other times in ns
+tau_curAP*   double - Time constant of active current at each compartment after action potential in
+pA
+amp_curAP*   double - Amplitude of active current at each compartment after action potential in pA
+
+g_sp         double - Conductance connecting soma and proximal dendrite, in nS.
+g_pd         double - Conductance connecting proximal and distal dendrite, in nS.
+t_ref        double - Duration of refractory period in ms.
+V_th         double - Spike threshold in mV.
+V_reset      double - Reset potential of the membrane in mV.
+V_max        double - Peak voltage at spike in mV
+jump_Th      double - Jump in adaptive threshold upon spike in mV
+tau_Th       double - Time constant for adaptive threshold in ms
+act_flag     double - flag to turn on calcium spikes
+reset_flag   double - flag to set somatic membrane potential to reset value at end of refractory
+period
+E_Ca         double - Reversal potential for calcium spike in mV
+G_Ca         double - Maximal conductance for calcium spike in nS
+tau_m        double - Time constant for activation M in ms
+tau_h        double - Time constant for inactivation H in ms
+half_m       double - Voltage for half activation of M-inf in mV
+half_h       double - Voltage for half activation of H-inf in mV
+slope_m      double - Slope of M-inf in mV-1
+slope_h      double - Slope of H-inf in mV-1
+
+Example:
+See examples/nest/mc_neuron.py.
+
+Remark:
+This is a prototype for illustration which has undergone only limited testing.
+Details of the implementation and user-interface will likely change.
+USE AT YOUR OWN PERIL!
```

Comment:
How correct is this? After all, this is code on which you have based a publication. Applies also to fixedca.